input()
raise Exception('Some exception')
